.travis.yml

```diff
@@ -31,13 +40,22 @@ cache:
 
 install:
   - ./.ci_scripts/start_docker.sh
-  - ./.ci_scripts/install_sdl.sh
+  - if [ "$BUILD_TARGET" != "appimage" ];
+    then
+      ./.ci_scripts/install_sdl.sh;
+    else
+      sudo apt-get -y install libgl1-mesa-dev libsdl2-dev libsdl2-mixer-dev;
+    fi;
 
 before_script:
   - ./.ci_scripts/run_cmake.sh
 
 script:
   - ./.ci_scripts/run_build.sh
+  - if [ "$BUILD_TARGET" == "appimage" ];
+    then
+      ./.ci_scripts/package_appimage.sh;
```

Comment:
Move this to `run_build.sh`: it's comparable to creating the `.app` file for macOS, which also lives in that script.